model/ios/registry.yaml

```diff
@@ -1,45 +1,43 @@
 groups:
-  - id: registry.ios.deprecated
+  - id: registry.ios
     type: attribute_group
-    display_name: Deprecated iOS Attributes
+    display_name: iOS Attributes
     brief: >
-        The iOS platform on which the iOS application is running.
+      The iOS platform on which the iOS application is running.
     attributes:
       - id: ios.state
```

Comment:
I know  it's been like this before, but `ios.state` naming does not seem quite right. It looks like it's a state of iOS and not the app.  If we have some other state on ios like `ios.wifi.state`, it would be even more confusing to have `ios.state` without any additional qualifier.

Maybe we can do `ios.app.state` and `andrioid.app.state` ?

Reply:
It's less confusing when it's simply scope to an event. 🙃